pydoctor/test/test_model.py

```diff
@@ -346,4 +346,15 @@ def test_c_module_python_module_name_clash(capsys:CapSys) -> None:
 
     finally:
         # cleanup
-        subprocess.getoutput(f'rm -f {package_path}/*.so')
\ No newline at end of file
+        subprocess.getoutput(f'rm -f {package_path}/*.so')
+
+def test_resolve_name_subclass(capsys:CapSys) -> None:
```

Comment:
can we have a docstring that describe what is tested here?

Maybe also have a test for sub-sub class inheritance  ... or maybe is not needed as we have good enough tests for our `Class.find` method.

```suggestion
def test_resolve_name_subclass(capsys:CapSys) -> None:
    """
    C{Model.resolveName} knows about single inheritance.
    """
```